docs/repo-docs/reference/configuration.mdx

```diff
@@ -339,8 +339,6 @@ An allowlist of environment variables that should be made available to this task
 
 A list of file glob patterns relative to the package's `package.json` to cache when the task is successfully completed.
 
-Omitting this key or passing an empty array tells `turbo` to cache nothing (except logs, which are always cached when caching is enabled).
```

Comment:
I move it down to add an example. So glob patterns is explained with an example, and then omitted key is explained with an example.

Reply:
Let's move this down below the existing example but not add another example. I trust that developers won't need an example of how to not write a key. 😄